counters/manager.go

```
@@ -141,24 +151,33 @@ func (m *ManagerStruct) dumpInfo(i *abstract.Info) {
 	manager.SaveInfo(i.ID, infoData)
 }
 
-func (m *ManagerStruct) loadInfo() {
+func (m *ManagerStruct) loadInfo() error {
 	manager := storage.GetManager()
 	var infoStruct abstract.Info
-	for _, infoData := range manager.LoadAllInfo() {
+	infos, err := manager.LoadAllInfo()
+	//FIXME: Should we panic?
+	if err != nil {
+		return err
+	}
+	for _, infoData := range infos {
 		json.Unmarshal(infoData, &infoStruct)
 		m.info[infoStruct.ID] = infoStruct
 	}
+	return nil
 }
 
-func (m *ManagerStruct) loadDomains() {
+func (m *ManagerStruct) loadDomains() error {
 	strg := storage.GetManager()
 	for key, info := range m.info {
 		switch info.Type {
 		case abstract.Default:
-			m.cache.Add(info.ID, hllpp.NewDomainFromData(info))
+			//FIXME: Do something with this error
+			domain, _ := hllpp.NewDomainFromData(info)
```

Should this error be ignored ?